cmd/migrate_from_mongo.go

```diff
@@ -19,10 +19,11 @@ import (
 )
 
 type MigrateFromMongoDBCmd struct {
-	MongoDB   commons.MongoDBConfig   `embed:"" prefix:"mongodb."`
-	Qdrant    commons.QdrantConfig    `embed:"" prefix:"qdrant."`
-	Migration commons.MigrationConfig `embed:"" prefix:"migration."`
-	IdField   string                  `prefix:"qdrant." help:"Field storing MongoDB IDs in Qdrant." default:"__id__"`
+	MongoDB     commons.MongoDBConfig   `embed:"" prefix:"mongodb."`
+	Qdrant      commons.QdrantConfig    `embed:"" prefix:"qdrant."`
+	Migration   commons.MigrationConfig `embed:"" prefix:"migration."`
+	IdField     string                  `prefix:"qdrant." help:"Field storing MongoDB IDs in Qdrant." default:"__id__"`
+	VectorField string                  `prefix:"mongodb." help:"Field to use as vector array."`
```

Comment:
Hey @superseb 
I believe Mongo supports multiple vector fields per document?
So this should be `[]string`.